src/status_im/subs/wallet/wallet.cljs

```diff
@@ -165,6 +170,22 @@
  :<- [:wallet/create-account]
  :-> :selected-keypair-uid)
 
+(rf/reg-sub
+ :wallet/selected-keypair
+ :<- [:wallet/keypairs]
+ :<- [:wallet/selected-keypair-uid]
+ (fn [[keypairs selected-keypair-uid]]
+   (some #(when (= (:key-uid %) selected-keypair-uid)
+            %)
+         keypairs)))
+
+(rf/reg-sub
+ :wallet/selected-primary-keypair?
+ :<- [:wallet/keypairs]
+ :<- [:wallet/selected-keypair-uid]
+ (fn [[keypairs selected-keypair-uid]]
+   (= selected-keypair-uid (:key-uid (first keypairs)))))
```

Comment:
IMO, we should check if the key pair type is `profile` for the primary/default key pair. The first keypair check could go wrong. WDYT?

Reply:
I checked it based on the previous implementation, I wasn't aware of that key meaning that!

Thank you @smohamedjavid ! I'll fix it.

Reply:
Thank you, now I fixed it.